tools/myst/host/package.c

```diff
@@ -128,23 +128,21 @@ int _package(int argc, const char* argv[])
     if ((argc < 4) || (cli_getopt(&argc, argv, "--help", NULL) == 0) ||
         (cli_getopt(&argc, argv, "-h", NULL) == 0))
     {
-        _err(USAGE_PACKAGE, argv[0], argv[0]);
+        _err_noexit(USAGE_PACKAGE, argv[0], argv[0]);
```

Comment:
Can you turn this into a macro in which "ret" is set to 1, just to match `_err` behavior?

Reply:
In #1351 I replaced `fprintf(stderr` with `_err(` a little too aggressively. `_err` has the advantage of printing the program name along with the error. However, I didn't realize that it also calls `exit(1)`. This PR is replacing most of those uses with `_err_noexit`, which does print the program name, but does not exit.